libraries/extractor/src/main/java/androidx/media3/extractor/metadata/id3/Id3Decoder.java

```diff
@@ -428,18 +429,24 @@ private static Id3Frame decodeFrame(
         String id = getFrameId(majorVersion, frameId0, frameId1, frameId2, frameId3);
         frame = decodeBinaryFrame(id3Data, frameSize, id);
       }
-      if (frame == null) {
-        Log.w(
-            TAG,
-            "Failed to decode frame: id="
-                + getFrameId(majorVersion, frameId0, frameId1, frameId2, frameId3)
-                + ", frameSize="
-                + frameSize);
-      }
-      return frame;
-    } finally {
+    } catch (Exception e) {
```

Comment:
What happens if this `catch Exception` isn't present (i.e. we only catch `OutOfMemoryError` here)?

Reply:
For this specific case the Exception catch is not necessary, I have some analytics logs about errors but no reproduction file provided by any users.

Reply:
This part if probably fine given we return null in other cases where we specifically detect that the frame is malformed.